timelock-impl/src/test/java/com/palantir/atlasdb/timelock/lock/watch/LockWatchingServiceImplTest.java

```diff
@@ -357,6 +370,11 @@ private static LockDescriptor descriptorForOtherTable() {
         return AtlasRowLockDescriptor.of(TABLE_2.getQualifiedName(), ROW);
     }
 
+    private static Optional<LockRequestMetadata> attachMetadataToLocks(Set<LockDescriptor> lockDescriptors) {
```

Comment:
nit: naming -- I'd maybe call this `createMetadataForLocks` instead? As you are not modifying the input (thus attaching)